sdk/storage/Azure.Storage.Files.Shares/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release History
 
+## 12.3.1 (2020-08-18)
+
+### Fixed
```

Comment:
We haven't used 3rd level headers anywhere else in our changelog.  Could we change this to regular text "Fix bug in TaskExtensions.EnsureCompleted method that causes it to unconditionally throw an exception in the environments with synchronization context"?